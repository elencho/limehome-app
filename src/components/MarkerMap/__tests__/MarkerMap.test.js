import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import MarkerMap from '../MarkerMap'

it("renders correctly", () => {
  const tree = renderer.create(<MarkerMap />).toJSON();
  expect(tree).toMatchSnapshot();
});
