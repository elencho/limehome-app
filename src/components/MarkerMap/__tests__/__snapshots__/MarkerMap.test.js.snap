// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<AIRMapMarker
  coordinate={
    Object {
      "latitude": undefined,
      "longitude": undefined,
    }
  }
  onPress={[Function]}
  stopPropagation={false}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "position": "absolute",
      },
      Object {
        "alignItems": "center",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#4A4B4D",
        "borderRadius": 5,
        "opacity": 1,
        "padding": 12,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 18,
        }
      }
    >
      15â‚¬
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#4A4B4D",
        "height": 10,
        "marginTop": -5,
        "transform": Array [
          Object {
            "rotate": "45deg",
          },
        ],
        "width": 10,
      }
    }
  />
</AIRMapMarker>
`;
